### PR TITLE
'technical measures' scope front and center

### PR DESCRIPTION
**Pull request by samuelweiler**
I propose to put the "technical measures" (or "means") scope restriction front and center.  This is purely a style thing, and I have no objection if you reject the change.

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h1 id="title">PROPOSED Private Advertising Technology Working Group Charter</h1
          motivated by the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>, is to
          specify web features and APIs that support advertising while
          acting in the interests of users, in particular providing strong
-         privacy assurances.  The Working Group welcomes participation from
+         privacy assurances using technical means.  The Working Group welcomes participation from
```
Comment (martinthomson) [inline charter.html:83]:
```suggestion
         privacy assurances using predominantly technical means.  The Working Group welcomes participation from
```

**Review by martinthomson — APPROVED** [review]
I'm OK with this.  The only problem is that it needs a little hedging, as I noted.

File: charter.html (hunk 1 above)
Comment (AramZS) [reply charter.html:83]: @samuelweiler I'm fine with this change if you are, and then I am fine merging.

Comment (samuelweiler) [reply charter.html:83]: fine with me

**Review by AramZS — APPROVED** [review]

**Pull request merged**